packages/network-of-terms-graphql/README.md

```diff
@@ -0,0 +1,198 @@
+# Network of Terms GraphQL API
+
+This application delivers a [GraphQL](https://graphql.org) API and command-line interface
+for searching the Network of Terms. It is available at https://termennetwerk-api.netwerkdigitaalerfgoed.nl.
+
+### GraphQL endpoint
+
+If you’re a developer and want to use the Network of Terms API in your apps,
+you can test the API and try GraphQL queries at the [API Playground](https://termennetwerk-api.netwerkdigitaalerfgoed.nl).
+
+You can connect your client apps to the [GraphQL endpoint](https://termennetwerk-api.netwerkdigitaalerfgoed.nl/graphql).
+
+### Host it yourself
+
+If you want to run the Network of Terms locally, or host it yourself, you can run our Docker image:
+
+    docker run -p 3123:3123 ghcr.io/netwerk-digitaal-erfgoed/network-of-terms-api
+
+and open http://localhost:3123 in your browser for the API Playground.
+
+While this repository follows [Semantic Versioning](https://semver.org), you may want to stick to a
+[tagged version](https://github.com/netwerk-digitaal-erfgoed/network-of-terms-api/releases):
+
+    docker run -p 3123:3123 ghcr.io/netwerk-digitaal-erfgoed/network-of-terms-api:v1.7.0
```

Comment:
Will we continue with the name `network-of-terms-api`? Or is it going to be replaced by `network-of-terms-graphql`?